### DISCUSSION TITLE
dark-themed Post page

### DISCUSSION THREAD

**Discussion by KTaha0** [dailydotdev/daily#1332] [Feature Requests]
When you click on the post, it takes you to a page that is brighter than Discord's bright mode. As programmers, we tend to prefer dark environments over bright ones, **so I suggest making the post page dark-themed and syncing it with the main page settings.**

**Comment by idoshamun** [comment]
Hi @MR-Unknown9,

Do you refer to the external page with the actual article?

**Reply by KTaha0** [reply]
yes, that's what I meant

**Reply by idoshamun** [reply]
as you can imagine we don't control the design of the external blog post

**Reply by KTaha0** [reply]
ooh...
I didn't know they were external, srry about that
